src/actions/breakpoints/remapLocations.js

```diff
@@ -2,7 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-export default function remapLocations(breakpoints, sourceId, sourceMaps) {
+ // @flow
+
+ export default function remapLocations(breakpoints:any, sourceId:any, sourceMaps:any) {
```

Comment:
Adding `:any` is easy but I think we can do better; you may want to dig into other files to get each of those arguments' true types.